velox/substrait/SubstraitParser.h

```diff
@@ -72,9 +72,11 @@ class SubstraitParser {
 
   /// Find the Velox function name according to the function id
   /// from a pre-constructed function map.
+  /// prefix: the prefix of registered SQL functions.
   std::string findVeloxFunction(
       const std::unordered_map<uint64_t, std::string>& functionMap,
-      uint64_t id) const;
+      uint64_t id,
+      const std::string& prefix = "") const;
```

Comment:
Do we need a default value here? If not, let's remove.

Reply:
Removed.